Kernel/Net/Routing.cpp

```diff
@@ -151,7 +153,9 @@ ErrorOr<void> update_routing_table(IPv4Address const& destination, IPv4Address c
         }
         if (update == UpdateTable::Delete) {
             for (auto& route : table) {
-                if (route == *route_entry) {
+                dbgln("candidate: {} {} {} {} {}", route.destination, route.gateway, route.netmask, route.flags, route.adapter);
```

Comment:
Too late, but this seems leftover from debugging.

Reply:
#14523